packages/synapse-react-client/src/style/components/_query-wrapper-plot-nav.scss

```diff
@@ -12,6 +12,10 @@
       grid-column: 1 / span 2;
     }
 
+    .ErrorBannerWrapper {
+      grid-column: 1 / span 2;
+    }
```

Comment:
Change to take up full width (if you get this error wrapper we probably don't have facet filters to show, so taking up full width is probably best)